EosPayload/drivers/radio_driver.py

```diff
@@ -89,12 +89,13 @@ def data_receive_callback(xbee_message):
             packet = xbee_message.data  # raw bytearray packet
             logger.info("Packet received ~~~~~~")
             logger.info(packet)
-            packet_object = Packet.decode(packet)  # convert packet bytearray to packet object
+            packet_object = Packet.decode(bytes(packet))  # convert packet bytearray to packet object
+            data_header = packet_object.data_header
 
             # Try to data log the packet, but we really don't want to block in a callback
             if self.log_lock.acquire(blocking=False):
                 try:
-                    self.data_log(["received", packet_object.encode_to_string()])
+                    self.data_log(["received"])
```

Comment:
change this to like "Received packet". Add this above it `# TODO change this to include packet as a string (encode_to_string function) `